packages/core/middleware/src/helpers.ts

```diff
@@ -0,0 +1,18 @@
+export const getAgnosticStatusCode = (obj: unknown, ...keys: string[]) => {
+  if (Array.isArray(obj)) {
+    for (const o of obj) {
+      if (getAgnosticStatusCode(o, ...keys)) {
+        return getAgnosticStatusCode(o, ...keys);
+      }
+    }
+  } else if (obj && typeof obj === 'object') {
+    for (const key of Object.keys(obj)) {
+      if (keys.includes(key)) {
```

Comment:
Would it be possible to do smth like?:
```ts
const variableNameExplainingUsWhatsItChecking = keys.includes(key);
```
Maybe it would make core more readable - currently I do not have idea why there is a condition like that and maybe with suggested approach I would